atd-vze/src/Components/GridExportDataButton.js

```diff
@@ -114,7 +119,7 @@ const GridExportDataButton = ({ query, columnsToExport, totalRecords, label }) =
 
     return cleanedAndFlattenedData;
   };
-// fa-2x ml-2 mt-1
+  // fa-2x ml-2 mt-1
```

Comment:
This could be removed since it looks like it was left behind sometime in the past. 🧹 🙏